src/Reflection/Annotations/AnnotationMethodReflection.php

```diff
@@ -28,6 +28,7 @@ public function __construct(
 		private array $parameters,
 		private bool $isStatic,
 		private bool $isVariadic,
+		private ?Type $throwType = null,
```

Comment:
Does not need to be optional

Reply:
I didn't know if i needed to be BC. Changed.